kubernetes/gaffer/values.yaml

```diff
@@ -66,6 +79,8 @@ ingress:
   enabled: true
   annotations:
     traefik.ingress.kubernetes.io/rule-type: PathPrefixStrip
+    route.openshift.io/termination: edge
+    haproxy.router.openshift.io/hsts_header: max-age=31536000;includeSubDomains;preload
```

Comment:
Do these changes belong in the core gaffer-docker module? What do they do?

Reply:
So this enables edge termination for tls and also forces any redirects which occur to be on the HTTPS protocol

Reply:
Are these openshift specific though? I am just not sure if openshift specific config should be in main gaffer-docker

Reply:
I think so yes. Where would you recommend they should be?

Reply:
I agree with @t92549. These are environment specific and shouldn't be in the default config for the Helm chart. Users should have their own `values.yaml` that configures the chart for their environment which they use when they deploy the chart.

Reply:
I know that the Traefik config is also environment specific, but we made an exception for that so that users could follow 'quick start' documentation without having to make too many config changes.